docs/api/associations.md

```diff
@@ -153,7 +153,7 @@ All methods return a promise
 
 <a name="hasmany"></a>
 ## `hasMany(target, [options])`
-[View code](https://github.com/sequelize/sequelize/blob/421f0f34356a3484b0f26e92e7fd133af6d3f6db/lib/associations/mixin.js#L245)
+[View code](https://github.com/sequelize/sequelize/blob/0c5c8bd7a208d8231ff9d138f369f287ae168cbe/lib/associations/mixin.js#L245)
```

Comment:
I really need to figure out a better solution to this so we don't have to commit sha hash for each method ...

I'm thinking we can add a preprocess step on RTD which inserts this instead